src/Microsoft.OData.Core/InstanceAnnotationWriteTracker.cs

```diff
@@ -6,46 +6,46 @@
 
 namespace Microsoft.OData
 {
-    using System;
-    using System.Collections.Generic;
+	using System;
+	using System.Collections.Generic;

```

Comment:
Could you revert changes to this file? It seems to introducing only indentation changes which are in themselves incorrect

Reply:
Done